lib/utils/formatTranslationChunks.js

```diff
@@ -0,0 +1,16 @@
+import { isArray } from 'lodash';
+
+/**
```

Comment:
Please remove the comment, we usually do not use it to explain what methods does. The code speaks for itself.